src/front-door/azext_front_door/_params.py

```diff
@@ -290,6 +292,7 @@ def load_arguments(self, _):
         c.argument('match_processing_behavior', arg_type=get_enum_type(MatchProcessingBehavior), help='Whether to stop processing rules after conditions in a rule is satisfied.')
 
     with self.argument_context('network front-door rules-engine rule action', arg_group='Forward Route Override') as c:
+        c.argument('action_type', arg_group="Action", arg_type=get_enum_type(['RequestHeader', 'ResponseHeader', 'ForwardRouteOverride', 'RedirectRouteOverride']), help='Action type to apply for a rule.')
```

Comment:
we still support these two. Which one is correct?

Reply:
It is still supported but in 'rules-engine rule create' command only header actions are supported as a starter rule.